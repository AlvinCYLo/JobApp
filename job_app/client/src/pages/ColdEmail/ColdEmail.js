import React from 'react';

const ColdEmail = () => {
  return <div>COLDEMAIL</div>;
};

export default ColdEmail;
