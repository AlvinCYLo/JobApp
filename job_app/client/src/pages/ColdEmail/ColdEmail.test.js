import React from 'react';
import renderer from 'react-test-renderer';
import ColdEmail from './ColdEmail';

it('renders correctly', () => {
  const tree = renderer.create(<ColdEmail />).toJSON();
  expect(tree).toMatchSnapshot();
});
