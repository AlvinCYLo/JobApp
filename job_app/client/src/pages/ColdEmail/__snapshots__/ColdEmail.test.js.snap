// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  COLDEMAIL
</div>
`;
